src/mongocxx/test/spec/monitoring.cpp

```diff
@@ -31,6 +33,33 @@ namespace spec {
 using namespace mongocxx;
 using bsoncxx::to_json;
 
+// commands postfixed with "_v2" are used to support the unified test format.
```

Comment:
Since the unified test format lists event monitoring commands in camelcase, I could either: try to mangle the new logic in with the old or create new functions altogether. It was simpler to create new functions, and it should be easier to get rid of old logic.